dbt/models/shared_columns.md

```diff
@@ -1347,6 +1353,42 @@ See the [model-sales-val](https://github.com/ccao-data/model-sales-val)
 repository for a list of possible flags.
 {% enddocs %}
 
+## sv_outlier_reason1
+
+{% docs shared_column_sv_outlier_reason1 %}
+One of three possible reasons that a sale is
+flagged as on outlier. The priority for
+sv_outlier_reason$n column filling is
+ptax outlier > price outlier > characteric outlier.
```

Comment:
**[Nitpick, non-blocking]** Small typo here, `characteric` should be `characteristic`. No suggestion for this change, since I include this fix in my bigger-picture suggestion below.